### ISSUE TITLE
Line-break after icons in mention links from Linear integration  

### ISSUE THREAD

**Issue by aspiers** [notion-enhancer/desktop#925]
**What is happening?**

When [the Notion <-> Linear integration](https://www.notion.so/integrations/linear) is enabled, notion-enhancer reformats any link to a Linear issue which is rendered as a "mention", so that there is an unnecessary line-break after the Linear icon.

**What should be happening?**

The formatting of the mention link should not change and there should be no extra line-break.

**Screenshots (if applicable):**

This is how it looks with notion-enhancer disabled (correct):

![image](https://github.com/user-attachments/assets/b4c335c8-c69c-4f7e-a33f-469aa9071115)

But when notion-enhancer is enabled, it looks like this **(incorrect)**:

![image](https://github.com/user-attachments/assets/570cf197-e620-4c84-94f2-a7c10120d365)

**Operating system/browser name & version:**

Linux (openSUSE Tumbleweed)

Chrome Version 123.0.6312.105 (Official Build) (64-bit)

**notion-enhancer and/or notion-repackaged version:**

notion-enhancer 0.11.1

**notion-enhancer configuration (enabled/disabled mods and their settings):**

I disabled _all_ extensions and themes.  I tried with "Load theme overrides" set to `Auto` / `Disabled` / `Enabled` and the problem was visible with all three.

**Comment by aspiers** [comment]
I traced the issue back to `display: block` being applied to the `<img>` element which displays the Linear icon.  It's caused by this CSS fragment from `@unocss-preflight-tailwind.css`:

![image](https://github.com/user-attachments/assets/724d8656-9c82-40b9-bfc2-0959895efbec)

Disabling that fragment fixes the issue.  It seems to me surprisingly invasive to blanket apply `display: block` to so many elements across the whole of Notion.  However I appreciate that this is coming from a third-party dependency:

https://github.com/notion-enhancer/notion-enhancer/blob/d5e0810915670f2ad7f1eefc9b87ac5838390e01/src/core/mod.json#L79

via f0e257044889326267fd23a1433e07434168fa4f, so maybe including this particular rule wasn't specifically chosen for inclusion?

**Comment by dragonwocky** [comment]
Yep, the preflight is useful for rendering the menu but the side effects across Notion itself are unintentional, I'll disable it in the main interface with the next patch. Thanks for the pickup.
